src/components/_inputs/InputNumber.vue

```diff
@@ -0,0 +1,9 @@
+<template>
+    <a-input-number
+      :name="name"
+      :min="minimum"
+      :max="maximum"
+      v-model="value"/>
```

Comment:
:stuck_out_tongue_winking_eye:

```suggestion
  <a-input-number
    :name="name"
    :min="minimum"
    :max="maximum"
    v-model="value"
  />
```